### DISCUSSION TITLE
In Collection view, dropdown fields display "value" instead of "text" of the options

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#14160] [Archive]
I found that in Collection view, dropdown fields display `value` instead of `text` of the options, which is not ideal IMO.

![image](https://user-images.githubusercontent.com/42278237/176230880-2372f71f-c8fe-4fd7-8f64-fd545af8ad1b.png)

While in Record view, it correctgly displays `text`.

![image](https://user-images.githubusercontent.com/42278237/176231096-cbd13e14-7c42-4a8d-8516-fe39f3aa160d.png)

I suggest to fix it if possible.

Thanks!

**Comment by rijkvanzanten** [comment]
Have you setup a Display or this field?

**Reply by powerseed** [reply]
No I didn't.

![image](https://user-images.githubusercontent.com/42278237/176281986-165becd2-f160-4144-8c8c-4979955175f3.png)

But I tried all 3 of them, none of them displayed the "text" I think.

**Reply by rijkvanzanten** [reply]
The Labels display has a similar "Show text X for value Y" configuration as the dropdown interface 👍🏻

**Reply by powerseed** [reply]
> The Labels display has a similar "Show text X for value Y" configuration as the dropdown interface 👍🏻

Ah... Got it. Thank you so much!
